cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -370,3 +370,27 @@ def test_migration_rbac(appliance, new_credential, v2v_providers):
         # Checks migration option is enabled in navigation
         assert 'Migration' in nav_tree['Compute'], ('Migration not found in nav tree, '
                                                     'rbac should allow this')
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal],],
```

Comment:
**REQUIRED**  :You can use `form_data_single_datastore` and avoid using `rhel7_minimal`.

Reply:
Done